spec/models/article_spec.rb

```diff
@@ -53,6 +53,26 @@ def build_and_validate_article(*args)
 
     it { is_expected.not_to allow_value("foo").for(:main_image_background_hex_color) }
 
+    describe "::admin_published_with" do
+      it "includes mascot published articles" do
+        user = create(:user, id: 1)
```

Comment:
The mascot won't always have an ID of 1. I believe I'm misunderstanding what we need here! Should there be logic to check `SiteConfig.mascot_user_id` both here and in the scope?

Reply:
Yup, this is correct. I fixed this test and overall logic.